dependencies.yaml

```
@@ -219,8 +222,16 @@ dependencies:
       - output_types: [conda, requirements, pyproject]
         packages:
           - cuda-python>=11.7.1,<12.0
```

We need `dask` in `test_python`, just not `distributed`.

See imports of `dask` in `python/benchmarks/single-node-io.py` and `python/tests/test_benchmarks.py`.
```suggestion
          - cuda-python>=11.7.1,<12.0
          - &dask dask >=2022.05.2
```